src/App.js

```diff
@@ -22,7 +23,8 @@ class App extends Component {
           <Route path="/calendar/" component={Calendar} />
           <Route path="/livefeed/" component={Livefeed} />
           <Route path="/experience-log/" component={ExperienceLog} />
-          <Route path="/exercises/" component={Exercises} />
+          <Route exact path="/exercises/" component={ExerciseList} />
+          <Route path="/exercises/:id" component={Exercise} />
```

Comment:
In other components, we delegate the sub-routing to avoid bloating the App component with a bunch of mixed concerns. See CompetenceOverview for an example of this delegation.